src/vs/workbench/browser/workbench.contribution.ts

```diff
@@ -113,7 +113,7 @@ const registry = Registry.as<IConfigurationRegistry>(ConfigurationExtensions.Con
 			},
 			'workbench.editor.languageDetectionHints': {
 				type: 'string',
-				default: 'always',
+				default: 'textEditors',
 				tags: ['experimental'],
 				enum: ['always', 'notebookEditors', 'textEditors', 'never'],
```

Comment:
An object `{ notebookEditors: true, textEditors: true }` would be more extensible to more than 2 editor types, if that would ever be relevant